data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Volgens Van Dale is *rails* gewoon meervoud. Een vorm die overigens [wel](http://nl.m.wiktionary.org/wiki/rail) kan is de *treinen rijden op een rail*. Rail is dan een synoniem voor het spoor en enkelvoud. Ik blijf erbij dat *rails* zonder lidwoord het beste is.